src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -623,7 +623,8 @@ trait OptimisticTransactionImpl extends TransactionalWrite with SQLMetricsReport
       numFilesTotal = postCommitSnapshot.numOfFiles,
       sizeInBytesTotal = postCommitSnapshot.sizeInBytes,
       protocol = postCommitSnapshot.protocol,
-      info = Option(commitInfo).map(_.copy(readVersion = None, isolationLevel = None)).orNull,
+      info = Option(commitInfo)
```

Comment:
Could you just remove `isolationLevel = None`? Could you also fix `isolationLevel = null` below?

Reply:
Done